apps/docs/content/components/installation.mdx

```diff
@@ -3,4 +3,26 @@ title: Installation
 description: The installation proccess
 ---
 
+import { installMethods } from "./data";
+
 # Installation
+
+## Install packages
+
+<Tabs tabs={installMethods}>
```

Comment:
If this gets reused a couple of times, we could extract a component that takes an array of packages + a scope

PackageManagerInstallation with props packages=["@hopper-ui/components"] and scope="prod"

and the pnpm/yarn/npm functions would be synced automatically. 

I already see in this example that PNPM saves to prod, but npm and yarn save to dev which is probably an issue

Reply:
Done